### PR TITLE
 added methods to control delay time and updated the README 

### PR DESCRIPTION
**Pull request by ninjaaron**
I thought it would be convenient to have methods for controlling the delay time.

### CODE REVIEW

File: src/Kaleidoscope-LEDEffect-Rainbow.h [hunk 1]
```diff
@@ -9,6 +9,7 @@ class LEDRainbowEffect : public LEDMode {
   LEDRainbowEffect(void) {}
 
   void brightness(byte);
+  void delay(byte);
```
Comment (algernon) [inline src/Kaleidoscope-LEDEffect-Rainbow.h]: Perhaps this should be called `update_delay` for consistency and clarity? `delay` alone tells little about what the setting is for.

File: README.md [hunk 2]
```diff
@@ -29,14 +29,18 @@ KALEIDOSCOPE_INIT_PLUGINS(LEDRainbowEffect, LEDRainbowWaveEffect);
...
 }
 ```
 
 ## Plugin methods
 
 The plugin provides two objects: `LEDRainbowEffect`, and `LEDRainbowWaveEffect`,
-neither of which have any public methods or properties, outside of those
-provided by all LED modes.
+both of which provide methods to set the delay time (animation speed)
+and effect brightness, called `delay` and `brightness` respectively.
```
Comment (algernon) [inline README.md]: Would it be possible for you to adjust this section to be more like in [other plugins](https://github.com/keyboardio/Kaleidoscope-OneShot#plugin-methods)?

**Review by algernon — CHANGES_REQUESTED** [review]
I agree with exposing the ability to set the update delay, thank you!

I left a few review notes - but I can do those changes myself too, if need be.

**Review by algernon — APPROVED** [review]

**Comment by algernon** [comment]
Much appreciated, thank you!

**Comment by ninjaaron** [comment]
No problem, and thank _you_ for all your contributions to the keyboardio firmware!

**Comment by ninjaaron** [comment]
Oh, and sorry about the leftover mistakes in the README! Thanks for fixing them for me.

**Comment by algernon** [comment]
No worries. They were easier to fix up than poke you to do it. =)

**Pull request merged**